cartridges/sfra_training/cartridge/scripts/steps/deleteCustomObjects.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var CustomObjectMgr = require('dw/object/CustomObjectMgr');
+var Transaction = require('dw/system/Transaction');
+
+/**
+ * @function deleteCOs
+ * @description Function that deletes all custom object for a CO type passed as a parameter.
+ *
+ * @param {Object} parameters Represents the parameters defined in the steptypes.json file
+ */
+module.exports = {
+    deleteCOs: function deleteCOs(parameters) {
+        var iterator = require('dw/util/Iterator');
```

Comment:
No need to import API Class Iterator cause CustomObjectMgr.queryCustomObjects returns the  instance of iterator

Reply:
Done1